agent/recognition/rdd_sampler.py

```
@@ -59,6 +76,11 @@ def __init__(self, default_dir_fn, option_fn=None):
             self._interval = float(os.environ.get("RDD_SAMPLE_INTERVAL", "1800"))
         except (TypeError, ValueError):
             self._interval = 1800.0  # 环境变量非法时兜底，避免 import 阶段崩溃
+        try:
+            self._max_files = int(os.environ.get("RDD_SAMPLE_MAX", str(_MAX_FILES)))
+        except (TypeError, ValueError):
+            self._max_files = _MAX_FILES
+        self._png_count = None       # 目录内 PNG 数，首次落盘时实扫一次，之后增量维护
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)